_build/partials/sandbox_about.hbs

```diff
@@ -4,19 +4,18 @@
       Planet Sandbox Data
       <img id="sandbox-planet-logo" src="{{rootUrl}}img/logos/planet-logo.svg" alt="">
     </h1>
-    <p> Test and experience a substantial amount of <a href="https://www.planet.com/">Planet</a> data samples made available under a <a href="https://creativecommons.org/licenses/by-nc/4.0/"> CC-BY-NC</a> (non-commercial) license to all active Planet users and Sentinel Hub users with a paid subscription. Sandbox Data offers a subset of the <a href="https://worldstrat.github.io/">WorldStrat</a> locations and dense time-stacks, ranging from 25 to 200 km2 in size. The locations are spread across the world, assuring a stratified representation of various use cases.</p>
+    <p> Test and experience a substantial amount of <a href="https://www.planet.com/" target="_blank">Planet</a> data samples made available under a <a href="https://creativecommons.org/licenses/by-nc/4.0/" target="_blank">CC-BY-NC</a> (non-commercial) license to all active Planet users and Sentinel Hub users with a paid subscription. Sandbox Data offers a subset of the <a href="https://worldstrat.github.io/" target="_blank">WorldStrat</a> locations and dense time-stacks, ranging from 25 to 200 km2 in size. The locations are spread across the world, assuring a stratified representation of various use cases.</p>
```

Comment:
Please also add `rel="noreferrer noopener"` wherever you added `target="_blank"`, this is for some security reasons to prevent tabnabbing (explained more in detail https://www.freecodecamp.org/news/how-to-use-html-to-open-link-in-new-tab/)

Reply:
I had it added at some point but then saw that modern Browser already include the "noopener" implicitly (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility). 

Otherwise `noreferrer` and `nooperne` seems to be redundant as `norefferer` includes `nooperne`. Anyway as most pages we link to, are internal, I think it will be interesting to have in the statistics included, that people came from the public collections page. As fat as I could see, not including noreferrer also doesn't pause any security risk.